vars/common.yml

```diff
@@ -1,11 +1,15 @@
 ---
 router:
   routes:
+    - dm-router-{env}.cloudapps.digital
+    - dm-router-{env}.cloudapps.digital/_metrics
     - www.{env}.marketplace.team
```

Comment:
This will get overwritten on production by the router block in:
https://github.com/alphagov/digitalmarketplace-aws/blob/master/vars/production.yml#L9

https://github.com/alphagov/digitalmarketplace-aws/blob/master/vars/README.md
> Values in a STAGE named file will override those in common.yml.

Reply:
Ah right yes - I was aware this PR was going to need some further screwing with, but hadn't noticed that one... had only worried about getting preview working to start with.